scoresheets/CleanTable.tex

```diff
@@ -2,18 +2,17 @@
 
 \begin{scorelist}
 	\scoreheading{Main Goal}
-	\scoreitem[5]{60}{Picking tableware and cutlery for transportation to the dishwasher}
-	\scoreitem[5]{180}{Placing the tableware and cutlery inside the dishwasher}
+	\scoreitem[5]{240}{Placing the tableware and cutlery inside the dishwasher}
 
 	\scoreheading{Bonus Rewards}
 	\scoreitem{200}{Opening the dishwasher door}
 	\scoreitem{100}{Pulling out the dishwasher rack}
-	\scoreitem[5]{40}{Placing an item correctly in the dishwasher}
-	\scoreitem{300}{Placing the dishwasher tab inside the dishwasher}
+	\scoreitem{200}{Placing all items correctly in the dishwasher}
+	\scoreitem{300}{Taking and placing the dishwasher tab inside the dishwasher}
```

Comment:
I think this is fair for bonus points.

I do wonder if the dishwasher tab can be placed anywhere in the dishwasher or if it has a small sepate compartment it must be placed in. That depends on the type of dishwasher and is not specified. For this many bonus points I would be fine with the latter.